qa/rpc-tests/disablewallet.py

```diff
@@ -2,10 +2,13 @@
 # Copyright (c) 2015-2016 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
+""" Various testcases for running a node with the -disablewallet option.
 
-#
-# Exercise API with -disablewallet.
-#
+Tests:
+    - validateaddress RPC doesn't cause bitcoind to crash when running with -disablewallet
+    (https://github.com/bitcoin/bitcoin/issues/6963#issuecomment-154548880)
```

Comment:
I think it is not required to be so specific in the doc string. You can leave this link where it was or remove it alltogether.

The test just checks if validateaddress works (even) with the wallet disabled. No need to mention "crash", imo.